### ISSUE TITLE
org.jooq.impl.ParserException with START WITH and ONLY clauses

### ISSUE THREAD

**Issue by cdxf** [jOOQ/jOOQ#7745]
### Steps to reproduce the problem (include example code if possible):
https://www.jooq.org/doc/3.11/manual/code-generation/codegen-ddl/
I tried to import sql file using DDLDatabase, the file contains `START WITH` and `ONLY` clauses which is generated from pg_dump.

Here is the errors:
```
org.jooq.impl.ParserException: Unexpected token or missing query delimiter: [110:5] ...tar_id integer
);



CREATE SEQUENCE sequence
    [*]START WITH 1
    INCREMENT BY 50
    NO MINVALUE
    NO MAXVALUE
    CACHE 1;



```
and
```

org.jooq.impl.ParserException: ADD, ALTER, COMMENT, DROP, MODIFY, or RENAME expected: [108:18] ...NULL,
    avatar_id integer
);


ALTER TABLE ONLY [*]avatar
    ADD CONSTRAINT avatar_pkey PRIMARY KEY (id);

```
### Versions:
- jOOQ:3.11
- Java:10
- Database (include vendor):Postgres
- OS:Windows
- JDBC Driver (include name if inofficial driver): postgresql

**Comment by srinivas-e212880** [comment]
Facing the same issue as below.
[ERROR] Failed to execute goal org.jooq.pro:jooq-codegen-maven:3.11.11:generate (default) on project ipr-jooq: Error running jOOQ code generation tool: Unexpected token or missing query delimiter: [317:2] ...-
[ERROR]
[ERROR] CREATE SEQUENCE GEN_ORG_TYPE_SEQ
[ERROR]         [*]START WITH     1
[ERROR]         INCREMENT BY   1
[ERROR]         NOCACHE
[ERROR]         NOCYCLE;

Using JOOQ: 3.11.5
DB: Oracle Flyway scripts

**Comment by knutwannheden** [comment]
@srinivas-e212880: As of jOOQ 3.12 the `CREATE SEQUENCE` statement you show is parsed correctly. I suggest you upgrade to jOOQ 3.12.1.

**Comment by srinivas-e212880** [comment]
> @srinivas-e212880: As of jOOQ 3.12 the `CREATE SEQUENCE` statement you show is parsed correctly. I suggest you upgrade to jOOQ 3.12.1.

Here I'm using JAVA 8. 
Is JOOQ-3.12.1 compatible with JAVA8?

**Comment by knutwannheden** [comment]
> Is JOOQ-3.12.1 compatible with JAVA8?

Yes. The jOOQ 3.12 Open Source edition is Java 8 compatible. For the commercial editions there are additionally also Java 6 and Java 11 builds available.

**Comment by srinivas-e212880** [comment]
> > Is JOOQ-3.12.1 compatible with JAVA8?
> 
> Yes. The jOOQ 3.12 Open Source edition is Java 8 compatible. For the commercial editions there are additionally also Java 6 and Java 11 builds available.

I'm using JOOQ Commercial edition only. 
Even upgraded to 3.12.1 (org.jooq.pro) with JAVA8 then facing incompatible issues as below.

[ERROR] Failed to execute goal org.jooq.pro:jooq-codegen-maven:3.12.1:generate (default) on project gen-jooq: Execution default of goal org.jooq.pro:jooq-codegen-maven:3.12.1:generate failed: Unable to load the mojo 'generate' in the plugin 'org.jooq.pro:jooq-codegen-maven:3.12.1' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: org/jooq/codegen/maven/Plugin has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

**Comment by knutwannheden** [comment]
@srinivas-e212880: You need to use the jOOQ artifacts with the Maven `groupId` `org.jooq.pro-java-8`. Your `pom.xml` should thus have a dependency like this:

```xml
            <dependency>
                <groupId>org.jooq.pro-java-8</groupId>
                <artifactId>jooq</artifactId>
                <version>3.12.1</version>
            </dependency>
```

The `org.jooq.pro` artifacts are indeed the Java 11 build.

Hope this helps.
